scania-digital-design-system/tegel
# fix(table): removing unused files and slots

## **Describe pull-request**  
There was a slot and file not used in the table so I have removed it.
The "no-result" slot was removed from the component back in May 2023, but its description remained in the file.
dummy-data.json was not used at all, and probably remained during migration from sdds.

## **Issue Linking:**  
- **No issue:** Found after support question relating to the no-result message.

## **How to test**  
1. Open preview link.
2. Check that "no-result" slot is not present under "Notes" of tds-table-body component

## **Checklist before submission**
